Resources/Prototypes/Recipes/Lathes/botany.yml

```diff
@@ -1,5 +1,5 @@
 - type: latheRecipe
-  id: MiniHoe
```

Comment:
Changes not related to your PR should be made in a separate PR

Reply:
removed all the renaming stuff